src/UIComponents/SelectTrainer.jsx

```diff
@@ -2,20 +2,28 @@
 import PropTypes from "prop-types";
 import React, { Component } from "react";
 import { connect } from "react-redux";
-import { setSelectedTrainer, getCompatibleTrainers } from "../redux";
+import { setSelectedTrainer, getCompatibleTrainers, setKValue } from "../redux";
 import { styles } from "../constants";
 
 class SelectTrainer extends Component {
   static propTypes = {
     selectedTrainer: PropTypes.string,
     setSelectedTrainer: PropTypes.func,
-    compatibleTrainers: PropTypes.object
+    compatibleTrainers: PropTypes.object,
+    setKValue: PropTypes.func, // set static propTypes for setKValue and kValue
```

Comment:
You can remove the comment here, please.

Reply:
done! 👍